app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team_list: list[Player]) -> int:
+    return sum([player.get_rating() for player in team_list])
```

Comment:
No need list convert
```suggestion
    return sum(player.get_rating() for player in team_list)
```